packages/terra-tabs/src/Tabs.scss

```diff
@@ -0,0 +1,105 @@
+@import './ModularTabs';
```

Comment:
Comment until decision is made?

Reply:
No need, none of that scss can be applied with the variant hardcoded to structural. .